Packs/GenericSQL/Integrations/GenericSQL/README.md

```diff
@@ -310,73 +312,93 @@ There is no context output for this command.
 
 ##### Context Example
+
 ```
 {}
 ```
 
 ##### Human Readable Output
+
 Command executed
 
 ## Troubleshooting
 
 ### General Test Connection Error
-In cases where you receive an error that is not clear when you **Test** the integration instance you can get detailed logs.
-1. Save the configured instance even though the **Test** doesn't work.
+
+In cases where you receive an error that is not clear when you __Test__ the integration instance you can get detailed logs.
+
+1. Save the configured instance even though the __Test__ doesn't work.
 2. In the playground, run the `!sql-command` with `debug-mode=true`. For example:
+
   ```
   !sql-command query="some simple query" debug-mode=true
   ```
+
 A log file will be generated in the Playground. Examine the log file for further details that explain why the integration is failing.
 
 ### Microsoft SQL Server
+
 We provide two options for connecting to Microsoft SQL Server:
-* **Microsoft SQL Server**: Uses the open source FreeTDS driver to communicate with Microsoft SQL Server. This driver supports authentication via domain logins (`DOMAIN\username`) with a password. If you do not require a domain login for authentication, we recommend using the `Microsoft SQL Server - MS ODBC Driver`.
-* **Microsoft SQL Server - MS ODBC Driver**: Official driver from Microsoft for Linux.
 
-**Note:** Kerberos authentication is not supported.
+- __Microsoft SQL Server__: Uses the open source FreeTDS driver to communicate with Microsoft SQL Server. This driver supports authentication via domain logins (`DOMAIN\username`) with a password. If you do not require a domain login for authentication, we recommend using the `Microsoft SQL Server - MS ODBC Driver`.
+- __Microsoft SQL Server - MS ODBC Driver__: Official driver from Microsoft for Linux.
+
+__Note:__ Kerberos authentication is not supported.
 
 If you experience any issues communicating with your Microsoft SQL Sever, try using both options as we've seen cases where one option works while the other doesn't.
 
+When configuring _SQL Server_, if you receive an error of the form:
 
-When configuring *SQL Server*, if you receive an error of the form:
 ```
 ('08S01', '[08S01] [FreeTDS][SQL Server]Unable to connect: Adaptive Server is unavailable or does not exist (20009) (SQLDriverConnect)')
 (Background on this error at: http://sqlalche.me/e/13/e3q8) 
-``` 
-It means there is a communication problem from the Generic SQL docker to the SQL Server. It usually means the dns hostname of the sql server is not resolving. You can try using an IP instead of the DNS. You can further test the from docker by running the following command on the Cortex XSOAR machine: 
+```
+
+It means there is a communication problem from the Generic SQL docker to the SQL Server. It usually means the dns hostname of the sql server is not resolving. You can try using an IP instead of the DNS. You can further test the from docker by running the following command on the Cortex XSOAR machine:
```

Comment:
```suggestion
It means there is a communication problem from the Generic SQL Docker to the SQL Server. It usually means the dns hostname of the SQL Server is not resolving. You can try using an IP instead of the DNS. You can further test from Docker by running the following command on the Cortex XSOAR machine:
```